components/Helpers/samples/Helpers.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Helpers
+author: 
+description: Various helpers to make conversions and other checks easier.
+keywords: Helpers, Theming, theme listerner, themes, screenunithelper, colorhelper
+dev_langs:
+  - csharp
+category: Helpers
+subcategory: Layout
+discussion-id: 0
+issue-id: 0
+---
+
+# ColorHelper
+
+The [ColorHelper](/dotnet/api/microsoft.toolkit.uwp.helpers.colorhelper) helps to convert various formats from and to `Color`.
+
+```csharp
+Color color = "#FFFF0000".ToColor();
+Color color = "Red".ToColor();
+string hex = Colors.Red.ToHex();
+HslColor hsl = Colors.Red.ToHsl();
+HsvColor hsv = Colors.Red.ToHsv();
+int i = Colors.Red.ToInt();
+```
+
+# ScreenUnitHelper
+
+The [ScreenUnitHelper](/dotnet/api/microsoft.toolkit.uwp.helpers.screenunithelper) helps to convert a screen unit to another screen unit (ex: 1cm => 39.7953px).
+
+```csharp
+float result = ScreenUnitHelper.Convert(ScreenUnit.Inch, ScreenUnit.Pixel, 1); // 96
+```
+
+# DesignTimeHelpers
+
+The [DesignTimeHelpers](/dotnet/api/microsoft.toolkit.uwp.ui.designtimehelpers) helps to detect if your code is running in execution or designtime mode.
+```csharp
```

Comment:
This needs an extra newline to render correctly:
![image](https://user-images.githubusercontent.com/9384894/235786725-06aa8a94-32e4-4fac-8cb4-5a2883f92a72.png)


```suggestion

```csharp
```